sql/src/main/java/com/vmware/ddlog/DDlogJooqProvider.java

```diff
@@ -117,22 +89,17 @@
     private final Map<String, Set<Record>> materializedViews = new ConcurrentHashMap<>();
     public static boolean trace = false;
 
-    public DDlogJooqProvider(final DDlogAPI dDlogAPI, final List<String> sqlStatements) {
+    public DDlogJooqProvider(final DDlogAPI dDlogAPI, final List<H2SqlStatement> sqlStatements) {
```

Comment:
do you want to also change the Javadoc for the class to indicate that it consumes the H2 dialect?